rpcserver.go

```diff
@@ -3074,6 +3190,27 @@ func (r *rpcServer) GetChanInfo(ctx context.Context,
 	return channelEdge, nil
 }
 
+func FindNodeChainInfo (chains *[]*lnrpc.NodeChainInfo, chain chainCode ) (*lnrpc.NodeChainInfo){
```

Comment:
Ditto here for similar logic to `FindPeerChain`, would be good to only have the logic in one place.